### ISSUE TITLE
Copying Strings

### ISSUE THREAD

**Issue by TBotV63** [meguy26/RemoteConsole#6]
`String` objects are immutable and thus must in 99.9% of all cases not be copied via
```java
new String(comm)
```

**Comment by meguy26** [comment]
Where was that? There are some places I do things that I didn't think I needed to, just to be sure.

**Comment by TBotV63** [comment]
`e.setComment(new String(comm));`
In your `SavePareser` class on line 194 for example

**Comment by meguy26** [comment]
Yeah, I remember that. I had done that because I was resetting the comm variable multiple times, but I suppose that wouldn't have an effect on the object itself.

**Comment by TBotV63** [comment]
Nope, no effect.
